package.json

```
@@ -35,6 +35,7 @@
   "dependencies": {
     "axios": "^0.24.0",
     "gluegun": "^5.0.0",
+    "regedit": "ironSource/node-regedit#f9df4db824fe68ec3b11e33af987ec42a2eeec32",
```

It looks like this package has been updated to publish types with the latest releases since the last review of this PR 🙌 

```suggestion
    "regedit": "5.1.1",
```